razorpay-payments.php

```diff
@@ -44,6 +51,10 @@ public function __construct()
             $this->key_secret = $this->settings['key_secret'];
             $this->payment_action = $this->settings['payment_action'];
 
+            $this->enable_webhook = $this->settings['enable_webhook'];
```

Comment:
We had that for payment_capture as well, and it broke. Can you try deleting the entry from your database and checking if it raises a notice?